admin_manual/configuration/server/occ_command.rst

```diff
@@ -867,15 +878,25 @@ The syntax is::
 
   group:list-members [options] <group>
 
-This example lists members of the Finance group:: 
+Options:
+
+::
+  
+  --output[=OUTPUT]  Output format (plain, json or json_pretty, default is plain) [default: "plain"]
+
+This example lists members of the Finance group
```

Comment:
s/This example lists members of the Finance group/This example lists members of the "Finance" group./g